src/ImportDefinitionsBundle/Form/Type/Interpreter/AssetsUrlInterpreterType.php

```diff
@@ -27,6 +28,7 @@ public function buildForm(FormBuilderInterface $builder, array $options)
     {
         $builder
```

Comment:
we could actually use the same form for both interpreters, what do you think?

Reply:
Agreed, I think we should consider deprecating `assets_url` completely and generalize into `array_interpreter` (or similar). It would allow to just call other (any) interpreters for each item in the list. I need this now since I need to use `object_resolver` for an array of items. Currently, it would mean adding `objects_resolver` which seems silly.

Maybe extend `nested` or you might have a better idea?

Reply:
this is actually sort of a similar thing than nested is. I would be happy to accept another PR for that.

Ok, so I will merge this PR as it is, and you will create another ok?